gocumber_test.go

```diff
@@ -201,3 +201,19 @@ func TestExec_NoMatch(t *testing.T) {
 
 	assert.False(t, steps.Exec("some unknown step"))
 }
+
+func TestDocstrings_OutlineVariables(t *testing.T) {
+	steps := make(Definitions)
+	tt := new(testing.T)
+
+	var called bool
+	steps.Given("something is:", func(_ []string, step StepNode) {
+		called = true
+		assert.Equal(t, "minimally functional\n\n", step.PyString().String())
```

Comment:
Oh, I didn't realize that `PyStringNode` stored separate lines. We need to keep doing that, so I guess iterate `Lines()` and `AddLine(replace(line))` many times.